src/engraving/style/defaultstyle.cpp

```diff
@@ -38,6 +39,7 @@ static const String LEGACY_MSS_V1_PATH(u":/engraving/styles/legacy-style-default
 static const String LEGACY_MSS_V2_PATH(u":/engraving/styles/legacy-style-defaults-v2.mss");
 static const String LEGACY_MSS_V3_PATH(u":/engraving/styles/legacy-style-defaults-v3.mss");
 static const String LEGACY_MSS_V302_PATH(u":/engraving/styles/legacy-style-defaults-v302.mss");
+static const String LEGACY_MSS_V450_PATH(u":/engraving/styles/legacy-style-defaults-v450.mss");
```

Comment:
Since this file was not added to `engraving.qrc`, it is not packaged with the application and thus will not be loaded. 

Does the fact that that was not noticed perhaps mean that we can also do without this file?